pycbc/workflow/core.py

```diff
@@ -213,8 +213,8 @@ def __init__(self, cp, name,
         # Determine the sections from the ini file that will configure
         # this executable
         sections = [name]
-        if self.ifo_string:
-            sec_tags = tags + [self.ifo_string]
+        if self.ifo_list is not None
+            sec_tags = tags + [self.ifo_list] + [self.ifo_string]
```

Comment:
The brackets around self.ifo_list should be removed, it is already a list and will raise an error. There is then an issue for single IFO running that self.ifo_list and [self.ifo_string] are identical, so we see the error:

>File "/home/c1239174/pycbc-dev/lib/python2.6/site-packages/PyCBC-88f5e1-py2.6.egg/pycbc/workflow/configuration.py", line 516, in check_duplicate_options
>    %(section1,section2,' '.join(duplicates)))
>ValueError: The following options appear in both section inspiral-h1 and inspiral-h1: h1-channel-name

I'd propose something more like:
```python
if self.ifo_list is not None:
    if len(self.ifo_list) > 1:
        sec_tags = tags + self.ifo_list + [self.ifo_string]
    else:
        sec_tags = tags + self.ifo_list
else:
    sec_tags = tags
```